src/libs/actions/Report.js

```diff
@@ -892,6 +892,71 @@ function deleteReportComment(reportID, reportAction) {
     API.write('DeleteComment', parameters, {optimisticData, successData, failureData});
 }
 
+/**
+ * Extract all links in a markdown comment and returns them in a list.
+ *
+ * @param {String} comment
+ * @returns {Array}
+ */
+const extractLinksInComment = (comment) => {
+    const reg = /\[[^[\]]*\]\(([^()]*)\)/gm;
+    const matches = [...comment.matchAll(reg)];
+    const links = _.map(matches, match => match[1]); // select the group from match
```

Comment:
Could you add a little more detailed comment, why exactly are we picking the first element of `match`?